Blog/app/controllers/comments_controller.rb

```diff
@@ -1,24 +1,24 @@
+# frozen_string_literal: true
+
 class CommentsController < ApplicationController
   def create
-    find_params
+    article
     @comment = @article.comments.create(comment_params)
     redirect_to article_path(@article)
   end
 
   def destroy
-    find_params
+    article
     @comment = @article.comments.find(params[:id])
```

Comment:
like in #create

Reply:
Changed this on the feature/suggestions branch which is waiting for being merged with this one.